packages/graphql-transformers-e2e-tests/src/__tests__/SubscriptionsRuntimeFiltering.e2e.test.ts

```diff
@@ -125,6 +125,36 @@ interface DeleteTaskInput {
   readOwners?: string[]
 }
 
+interface CreateTaskGroupInput {
```

Comment:
Since these interfaces have same properties, can we have one `TaskGroup` base interface and use it everywhere or extend it if needed?